spring-web/src/test/java/org/springframework/http/converter/ResourceHttpMessageConverterTests.java

```diff
@@ -156,5 +156,123 @@ public void writeContentInputStreamThrowingNullPointerException() throws Excepti
 
 		assertThat(outputMessage.getHeaders().getContentLength()).isEqualTo(0);
 	}
+	@Test
```

Comment:
```suggestion

	@Test
```
checkstyle solution seems broken and costly to obtain such a custom tool not working. better use spot.